### PR TITLE
FIX : Implement Risk Indicators To Review and Deploy Modal

### PR DESCRIPTION
**Pull request by sticklo**
## What does this PR do and why?

Added Risk Indicators to necessary settings in the "Review and Deploy" Modal

## Screenshots or screen recordings
**AFTER-FIX**
image

## Acceptance checklist

- [x] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [x] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).

### CODE REVIEW

File: src/components/v1/V1Create/ConfirmDeployProject.tsx [hunk 1]
```diff
@@ -51,8 +56,13 @@ export default function ConfirmDeployProject() {
     editingFC?.currency.toNumber() as V1CurrencyOption,
   )
 
-  const rowGutter: [Gutter, Gutter] = [25, 20]
+  const unsafeFundingCycleProperties = getUnsafeFundingCycleProperties(
+    currentFC as V1FundingCycle,
+  )
+  console.warn(unsafeFundingCycleProperties)
```
[the remark this answers is not in the document]
Comment (sticklo) [reply src/components/v1/V1Create/ConfirmDeployProject.tsx]: Fixed! Thanks for the feedback

[the remark this answers is not in the document]
Comment (sticklo) [reply src/components/v1/V1Create/ConfirmDeployProject.tsx]: Hold on, my bad let me fix the spacing thing you mentioned

[the remark this answers is not in the document]
Comment (johnnyd-eth) [reply src/components/v1/V1Create/ConfirmDeployProject.tsx]: Great! Let's just fix up these merge conflicts then we'll be good to go 🚀

[the remark this answers is not in the document]
Comment (sticklo) [reply src/components/v1/V1Create/ConfirmDeployProject.tsx]: oh so sorry i missed your comments, would resolve and update my PR as soon as possible, thank you

**Review by johnnyd-eth — APPROVED** [review]

**Pull request merged**